docs/hub/billing.md

```diff
@@ -1,30 +1,75 @@
+</div>
+
+### Available payment methods
+
+Hugging Face uses [Stripe](https://stripe.com) to securely process your payment information.
+
+The only payment method supported for Hugging Face compute services is credit cards.
+You can add a credit card to your account from your billing settings.
+
+<div class="flex justify-center">
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/payment-method-light.png"/>
+	<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/payment-method-dark.png"/>
+</div>
+
+You also have an option to link your Hugging Face organization to your AWS account via [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-n6vsyhdjkfng2). Hugging Face compute service usage will then be included in your AWS bill.
+
+Read more in our [blog post](https://huggingface.co/blog/aws-marketplace).
+
+### Billing thresholds & Invoicing
+
+When using credit cards as a payment method, you'll be billed for the Hugging Face compute usage each time the accrued usage goes above a billing threshold for your user or organization.
+
+On the 1st of every month, Hugging Face edits an invoice for usage accrued during the prior month. Any usage that has not yet been charged yet will be charged at that time.
+
+For example, if your billing threshold is set at $100.00, and you incur $254.00 of usage during a given month, your credit card will be charged a total of three times during the month:
+- Once for usage between $0 and $100: $100
+- Once for usage between $100 and $200: $100
+- Once at the end of the month for the remaining $54: $54  
+
+Note: this will be detailed in your monthly invoice.
 
 <div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/billing.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/billing-dark.png"/>
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/explain-threshold.png "/>
 </div>
 
-Any feedback or support request related to billing is welcome at billing@huggingface.co.
+You can view receipts and invoices from your billing dashboard.
 
-## Invoicing
+<div class="flex justify-center">
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/threshold-payments-light.png "/>
+	<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/threshold-payments-dark.png"/>
+</div>
+
+## Enterprise Hub subscriptions
```

Comment:
Should we mention PRO as well?

Reply:
Yes, I added a blurb at the end of the document